doc/external-signer.md

```diff
@@ -37,7 +37,7 @@ The master key fingerprint is used to identify a device.
 Create a wallet, this automatically imports the public keys:
 
 ```sh
-$ bitcoin-cli createwallet "hww" true true "" true true true
+$ bitcoin-cli -named createwallet wallet_name="hww" disable_private_keys=true blank=true passphrase="" avoid_reuse=true descriptors=true load_on_startup=true external_signer=true
```

Comment:
You should be able to drop most of these named arguments, except `wallet_name` and `external_signer`. Most are already defaults, except `avoid_reuse` but that's not relevant to this tutorial.

Reply:
New version pushed with `blank`, `avoid_reuse` and `load_on_startup` removed.